plugins/cilium-cni/cni-install.sh

```diff
@@ -83,6 +83,23 @@ if [ "${CILIUM_CUSTOM_CNI_CONF}" = "true" ]; then
 	exit 0
 fi
 
+
```

Comment:
Seems like this commit can just be squashed with the previous commit.

Reply:
Done!